registry/1000g-subset.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "1000g-subset",
+    "description": "Subset of 1000G Low Coverage WGS Data",
+    "target": "1000g-subset/reads",
```

Comment:
Deffo; will make the change. I've already pushed the data into S3; I'll move it there as well.